SharpBucket/V2/EndPoints/PullRequestResource.cs

```diff
@@ -112,5 +112,10 @@ public Comment GetPullRequestComment(int commentId)
         {
             return _repositoriesEndPoint.GetPullRequestComment(_accountName, _repository, _pullRequestId, commentId);
         }
+
+        public Comment PostPullRequstComment(Comment comment)
```

Comment:
There is a typo issue in the name of that method
```suggestion
        public Comment PostPullRequestComment(Comment comment)
```